cmd/experimental/kjobctl/apis/v1alpha1/application_profile_types.go

```diff
@@ -93,6 +94,7 @@ type TemplateReference string
 // +kubebuilder:validation:XValidation:rule="!has(self.requiredFlags) || !('ntasks' in self.requiredFlags) || self.name == 'Slurm'", message="ntasks flag can be used only on Slurm mode"
 // +kubebuilder:validation:XValidation:rule="!has(self.requiredFlags) || !('output' in self.requiredFlags) || self.name == 'Slurm'", message="output flag can be used only on Slurm mode"
 // +kubebuilder:validation:XValidation:rule="!has(self.requiredFlags) || !('partition' in self.requiredFlags) || self.name == 'Slurm'", message="partition flag can be used only on Slurm mode"
+// +kubebuilder:validation:XValidation:rule="!has(self.requiredFlags) || !('time' in self.requiredFlags) || self.name == 'Slurm'", message="time flag can be used only on Slurm mode"
```

Comment:
Is this flag slurm only for real or just slurp to begins with?
Will other modes also get it in the next step?

Reply:
I think we can add to other modes in follow-ups. But I'm not sure that it should be named as "time".

Reply:
Created an issue https://github.com/kubernetes-sigs/kueue/issues/3387.